services/horizon/internal/ingest/processor_runner.go

```diff
@@ -206,7 +206,7 @@ func (s *ProcessorRunner) RunHistoryArchiveIngestion(checkpointLedger uint32) (i
 		}
 	} else {
 		if err = s.checkIfProtocolVersionSupported(checkpointLedger); err != nil {
-			return changeStats.GetResults(), errors.Wrap(err, "Protocol version not supported")
+			return changeStats.GetResults(), errors.Wrap(err, "Error checking protocol version is supported")
```

Comment:
The change makes sense, but maybe we can word it more clearly?
```suggestion
Error while checking for supported protocol version (<= 15)
```
Where the reported number is a variable